libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/tabs/reviewCreateTab.tsx

```diff
@@ -0,0 +1,68 @@
+import type { RootState } from '../../../../../core/state/templates/store';
+import { useSelector } from 'react-redux';
+import { useIntl, type IntlShape } from 'react-intl';
+import { Button, Spinner } from '@fluentui/react-components';
+import constants from '../../../../../common/constants';
+import { useState } from 'react';
+
+export const ReviewCreatePanel = (
+  // { onCreateClick }: { onCreateClick: () => Promise<void> }
+) => {
+  const intl = useIntl();
+  const { workflowName, kind } = useSelector((state: RootState) => state.template);
+  const { workflowName: existingWorkflowName } = useSelector((state: RootState) => state.workflow);
```

Comment:
I would suggest you to name this as existingWorkflowName in the workflow store to keep it different from template and avoid confusion.